Examples/r/live_objects/example.cxx

```diff
@@ -0,0 +1,20 @@
+/* File : example.cxx */
+
+#include "example.h"
+
+unsigned Foo::count = 0;
+
+Foo::Foo()
+{
+  ++count;
+}
+
+Foo::~Foo()
+{
+  --count;
+}
+
+unsigned Foo::get_count()
+{
+  return count;
+};
```

Comment:
No newline at end of file.  Also a superfluous `;` after a function's closing `}`.